controllers/users.js

```diff
@@ -34,11 +34,11 @@ var error400 = function(msg, callback) {
   });
 };
 
-var getType = function(user, admins) {
+var getType = function(user) {
   if (user.roles && user.roles.length) {
     return user.roles[0];
   }
-  return admins[user.name] ? 'admin' : 'unknown';
```

Comment:
Looks right to me. Worst case we'll mark some admins as unknown. I think it's just a label in the UI.